suite/tests/client-interface/low_on_memory.c

```diff
@@ -0,0 +1,61 @@
+/* **********************************************************
+ * Copyright (c) 2011-2019 Google, Inc.  All rights reserved.
+ * Copyright (c) 2008-2010 VMware, Inc.  All rights reserved.
```

Comment:
If you wrote this from scratch no need for this

Reply:
Done